Makefile

```diff
@@ -99,6 +100,8 @@ define install_acrn_debug
 endef
 
 HV_MAKEOPTS := -C $(T)/hypervisor BOARD=$(BOARD) SCENARIO=$(SCENARIO) HV_OBJDIR=$(HV_OUT) RELEASE=$(RELEASE)
+board_inspector:
+	@python3 misc/packaging/gen_acrn_deb.py $(ROOT_OUT) $(FULL_VERSION) board_inspector
```

Comment:
This seems odd to call the same ``gen_acrn_deb.py`` script with a different number of positional arguments.  This one explicitly says "board_inspector" but the other one doesn't say "hypervisor", for example.

Reply:
Fixed.